build-ci-wheels/action.yml

```
@@ -0,0 +1,77 @@
+name: "Build a C-extension Python library"
+description: "Build a C-extension Python library by generating manylinux wheels"
+
+inputs:
+  cpython-version:
+    description: 'Desired CPython version'
+    default: '3.10'
+    required: false
+    type: string
+  cibw-build:
+    description: 'Desired build wheel CPython version'
+    default: "cp310-*"
+    required: false
+    type: string
+  cibw-skip:
+    description: 'Desired build wheel for skipping'
+    default: "*-musllinux* "
+    required: false
+    type: string
+  cibw-archs:
+    description: 'Desired build arhitecture'
+    default: " auto "
```

Should we get rid of these extra spaces?

```suggestion
    default: "auto"
```